auth/iam_cache.go

```diff
@@ -36,14 +36,14 @@ type IAMCache struct {
 var _ IAMService = &IAMCache{}
 
 type item struct {
-	value Account
 	exp   time.Time
+	value Account
 }
 
 type icache struct {
-	sync.RWMutex
-	expire time.Duration
 	items  map[string]item
+	expire time.Duration
+	sync.RWMutex
```

Comment:
I think we should still follow some good style guidelines here.  Embedded structs should still go first (https://github.com/uber-go/guide/blob/master/style.md#embedding-in-structs), and mutex should precede what they protect